src/commands/utility/ping.js

```diff
@@ -0,0 +1,10 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('ping')
+		.setDescription('Replies with Pong!'),
+	async execute(interaction) {
+		await interaction.reply('Pong!');
+	},
+};
```

Comment:
Various file needs the EOF line
![image](https://github.com/user-attachments/assets/3031ced4-25ec-4d48-9ce2-3dbe76810311)